code/__DEFINES/modular_computer.dm

```diff
@@ -17,6 +17,10 @@
 #define PROGRAM_CONSOLE (1<<0)
 #define PROGRAM_LAPTOP (1<<1)
 #define PROGRAM_TABLET (1<<2)
+#define PROGRAM_NON_SILICON (PROGRAM_ALL & ~PROGRAM_SILICON)
+#define PROGRAM_CYBORG (1<<3)
+#define PROGRAM_AI (1<<4)
+#define PROGRAM_SILICON (PROGRAM_CYBORG | PROGRAM_AI)
```

Comment:
I am worried about the implications of limiting apps to silicons & non-silicons, they ARE tablets and SHOULD be sharing apps together, there's an entire feature based entirely around that through program disks

Reply:
there are gonna be some apps that you won't have on normal tablets, like robotact
you could argue that robotact is not a normal app and it should not be available in ntdownloader, and that'd be fair and it would make these defines useless tho

Reply:
I don't think this needs some unique define, the app can just be not-transferable and not available on ntnet

Reply:
fair enough